staging/src/k8s.io/client-go/rest/client.go

```diff
@@ -145,12 +167,13 @@ func readExpBackoffConfig() BackoffManager {
 	backoffBaseInt, errBase := strconv.ParseInt(backoffBase, 10, 64)
 	backoffDurationInt, errDuration := strconv.ParseInt(backoffDuration, 10, 64)
 	if errBase != nil || errDuration != nil {
-		return &NoBackoff{}
+		return defaultBackoffManager()
 	}
 	return &URLBackoff{
-		Backoff: flowcontrol.NewBackOff(
+		Backoff: flowcontrol.NewBackOffWithJitter(
 			time.Duration(backoffBaseInt)*time.Second,
-			time.Duration(backoffDurationInt)*time.Second)}
+			time.Duration(backoffDurationInt)*time.Second,
+			defaultBackoffMaxJitter)}
```

Comment:
shouldn't this be configurable too?

Reply:
i am not sure that it matters a lot, but i am fine making it configurable too. any thoughts here @wojtek-t?